python/lsst/ip/isr/ptcDataset.py

```diff
@@ -151,8 +151,6 @@ class PhotonTransferCurveDataset(IsrCalib):
     `lsst.cp.pipe.ptc.PhotonTransferCurveDataset`
```

Comment:
This returns section also needs to be removed (because otherwise the Version 1.1 text is in the wrong place and not part of the Notes).